Utilities/Dockers/Alphashape/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get -qq update
+RUN apt-get -qq -y install automake
+RUN apt-get -qq -y install build-essential
+RUN apt-get -qq -y install bzip2
+RUN apt-get -qq -y install curl
+RUN apt-get -qq -y install git
+RUN apt-get -qq -y install libbz2-dev
+RUN apt-get -qq -y install libcurl4-openssl-dev
+RUN apt-get -qq -y install liblzma-dev
+RUN apt-get -qq -y install libncurses-dev
+RUN apt-get -qq -y install libncurses5-dev
+RUN apt-get -qq -y install python3
+RUN apt-get -qq -y install python3-dev
+RUN apt-get -qq -y install python3-distutils
+RUN apt-get -qq -y install python3-pip
+RUN apt-get -qq -y install python3-setuptools
+RUN apt-get -qq -y install wget
+RUN apt-get -qq -y install zlib1g-dev
+
+ENV SRC=/usr/local/src
+ENV BIN=/usr/local/bin
```

Comment:
it doesn't look like `$BIN` is used anywhere.

Reply:
Removed.